Source/WebGPU/WGSL/ConstantValue.h

```diff
@@ -150,6 +166,21 @@ struct ConstantValue : BaseValue {
             return *abstractInt;
         RELEASE_ASSERT_NOT_REACHED();
     }
+    half getHalf() const
+    {
+        if (auto* i32 = std::get_if<int32_t>(this))
+            return *i32;
+        if (auto* u32 = std::get_if<uint32_t>(this))
+            return *u32;
+        if (auto* abstractInt = std::get_if<int64_t>(this))
+            return *abstractInt;
+        if (auto* f32 = std::get_if<float>(this))
+            return *f32;
+        if (auto* f64 = std::get_if<double>(this))
+            return *f64;
+        if (auto* b = std::get_if<bool>(this))
+            return *b;
```

Comment:
You'll probably need a `RELEASE_ASSERT_NOT_REACHED();` here to make the compiler happy

Reply:
Done.